app/controllers/api/v1/rental_assistances_controller.rb

```diff
@@ -34,8 +34,7 @@ def update
         )
 
         if response
-          updated_rental_assistances = soql_rental_assistance_service.application_rental_assistances(params[:application_id])
```

Comment:
Do you remember if this was something that was supposed to be in or out? Seems helpful just wasn't clear on the motivation